src/Microsoft.Framework.Runtime.Interfaces/IAssemblyLoadContextAccessor.cs

```diff
@@ -2,9 +2,21 @@
 
 namespace Microsoft.Framework.Runtime
 {
+    /// <summary>
+    /// Provides access to created <see cref="IAssemblyLoadContext"/>
+    /// </summary>
     public interface IAssemblyLoadContextAccessor
     {
+        /// <summary>
+        /// Gets the default <see cref="IAssemblyLoadContext"/>.
```

Comment:
`default` => `current`

I'd actually suggest renaming the property too... Maybe I'm misunderstanding the accessor's purpose though.